pkgs/development/python-modules/pyparser/default.nix

```diff
@@ -0,0 +1,27 @@
+{ buildPythonPackage, lib, fetchFromBitbucket
+, parse
+}:
+
+buildPythonPackage rec {
+  pname = "pyparser";
+  version = "1.0";
+
+  src = fetchFromBitbucket {
+    owner = "rw_grim";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0aplb4zdpgbpmaw9qj0vr7qip9q5w7sl1m1lp1nc9jmjfij9i0hf";
+  };
+
+  # strictly requires 1.6.5 but nix has 1.6.6 only
+  preConfigure = "sed -i 's/1.6.5/1.6.6/' requirements.txt";
```

Comment:
`prePatch`
I think you should rather replace `==` by `>=` to make this patch future-proof.

Reply:
yeah that was stupid by me, fixing

Reply:
I'm not on bitbucket and I'm not willed to sign up, and I could not find  his mail address. He is @grimmy on github